### PR TITLE
Issue three can start car

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-02-18 at 6 03 33 PM

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,13 +1,40 @@
 var $car = document.querySelector('.car-image');
+$car.style.top = '0';
+$car.style.left = '0';
+var x = 0;
+var y = 0;
+
+var carData = [
```
Comment (codingcodymiller) [inline main.js]: You don't need this in an array, it can just be the object stored within the array

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ameliatalkington** [comment]
Screen Shot 2021-02-18 at 6 23 35 PM

File: main.js [hunk 2]
```diff
@@ -1,13 +1,38 @@
 var $car = document.querySelector('.car-image');
+$car.style.top = '0';
+$car.style.left = '0';
+var x = 0;
+var y = 0;
```
Comment (codingcodymiller) [inline main.js]: You can remove these, you should use the properties in your data model

File: main.js [hunk 3]
```diff
@@ -1,13 +1,38 @@
...
+  direction: 'east',
+  location: {
+    x: 0,
+    y: 0
+  }
+};
+
+function moveCar() {
+  translate = 'translate(' + x + 'px, ' + y + 'px)';
+  x = x + 7;
```
Comment (codingcodymiller) [inline main.js]: You should update `carData.x` instead

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ameliatalkington** [comment]
Screen Shot 2021-02-18 at 6 31 58 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**